### PR TITLE
created reusable signup screen container

### PR DESCRIPTION
**Pull request by nicholasrobinharrison**
new components folder with a signup container, just put whatever you need to inside as you would any other component ex: 

import SignUpContainer from '../components'

return (
    '<' SignUpContainer '>'
      * Stuff goes in here*
    '<'/SignUpContainer'>'
)

could be refactored to receive props, if we need to

### CODE REVIEW

File: client/src/components/SignUpContainer.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    padding: "5vh",
+    textAlign: "center",
+    rounded: true,
+    backgroundColor: "white",
+    margin: "auto"
+  }
+};
+
+const SignUpContainer = props => {
+  const classes = useStyles;
```
Comment (skassam21) [inline client/src/components/SignUpContainer.js]: we should actually use the hooks API here, so notice the example using the makeStyles function around useStyles

File: client/src/components/SignUpContainer.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { Link } from "react-router-dom";
+import { Button, Paper, TextField, Typography, Grid } from "@material-ui/core";
+
+const useStyles = {
+  root: {
+    padding: "10vh",
+    backgroundColor: "#6E3ADB",
+    height: "80vh"
+  },
+  text: {
```
Comment (skassam21) [inline client/src/components/SignUpContainer.js]: This is not used

**Review by skassam21 — APPROVED** [review]
Looks great overall, some small things

Just fix them and then you can merge

**Review by rkerins91 — COMMENTED** [review]
This is great. The one suggestion I have is to make it scrollable (overflowY: "auto") and have a max height. When I add a bunch of stuff, it extends the component past the background like in the picture that I included, and the user could end up putting down a lot of languages as part of their experience.
Screen Shot 2020-02-10 at 4 54 37 PM

File: client/src/components/SignUpContainer.js (hunk 1 above)
Comment (nicholasrobinharrison) [reply client/src/components/SignUpContainer.js]: when I use make styles I'm getting this error "The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX." I'm not sure what I'm doing differently from the example

Comment (nicholasrobinharrison) [reply client/src/components/SignUpContainer.js]: when I log out the classes object it look like this {root: "makeStyles-root-1", signUp: "makeStyles-signUp-2"}, so the makeStyles hook is converting these object into strings

Comment (skassam21) [reply client/src/components/SignUpContainer.js]: That's right, so the nuance is below you are using `style` as the HTML attribute, but you should be using `className` if you are using the hooks

**Review by skassam21 — COMMENTED** [review]
On another note, would be cool to do the gradient background

**Review by rkerins91 — APPROVED** [review]

**Pull request merged**